packages/forms/src/elements/MultiThumbRange.tsx

```diff
@@ -28,382 +19,101 @@ import {
 } from '../styled';
 import useFieldContext from '../utils/useFieldContext';
 
+const MIN = 0;
+const MAX = 100;
+
 /**
  * @extends HTMLAttributes<HTMLDivElement>
  */
 export const MultiThumbRange = forwardRef<HTMLDivElement, IMultiThumbRangeProps>(
-  ({ min, max, minValue, maxValue, disabled, step, onChange, onMouseDown, ...props }, ref) => {
+  (
+    {
+      min = MIN,
+      max = MAX,
```

Comment:
Do we need to define a default value for `min` and `max` here if the `defaultProps` were set?

Reply:
This helps avoid type-checked conditional logic for calculations that follow.